loopchain/blockchain/votes/v1_0/vote_factory.py

```diff
@@ -70,9 +70,9 @@ async def create_vote(self, data_id: bytes, commit_id: bytes, epoch_num: int, ro
         return vote
 
     def create_none_vote(self, epoch_num: int, round_num: int) -> BlockVote:
-        timestamp = util.get_time_stamp()
+        timestamp = utils.get_time_stamp()
         signature = self._get_signature(
-            height=0,
+            height=-1,
```

Comment:
> Possible related?: https://github.com/icon-project/loopchain/pull/512/files#r464295994

Can you explain why you set the height `-1`

Reply:
It will be recovered.